CHANGELOG.md

```diff
@@ -4,6 +4,59 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.48.1] - 2024-06-10
+
+## 🐛 Fixes
+
+### Improve error message produced when a subgraph response doesn't include an expected `content-type` header value ([Issue #5359](https://github.com/apollographql/router/issues/5359))
+
+To improve a common debugability challenge when a subgraph response doesn't contain an expected `content-type` header value, the error message produced will include additional details about the error.
```

Comment:
```suggestion
To improve a common debuggability challenge when a subgraph response doesn't contain an expected `content-type` header value, the error message produced will include additional details about the error.
```

Reply:
Hah.  My editor told me it's one `g`.  I've previously used two `g`s.  I don't know if there is a right answer and Google has conflicting feelings, but sure, `gg` sounds good. ;)